docs/commands.md

```diff
@@ -456,6 +456,12 @@ This section contains information on all supported functions from the Cypher que
 | ------- |:-----------|
 |exists() | Returns true if the specified property exists in the node or relationship. |
 
+## Node functions
+|Function | Description|
+| ------- |:-----------|
+|indegree() | Returns the number of node's incoming edges. |
+|outdegree() | Returns the number of node's outgoing edges. |
```

Comment:
We should add some documentation on how these functions can optionally filter by relationship type.

Reply:
Agree, I think each function on this list should get its own section
@K-Jo is there any chance of us getting help extending our documentation from probably Ben?